packages/aws-rfdk/docs/upgrade/upgrading-1.5.md.yaml

```diff
@@ -0,0 +1,13 @@
+# Upgrading to RFDK v1.5.x or Newer
```

Comment:
Is this file type correct?  I see it's a `.md.yaml`, should it just be `.md` instead?

Reply:
Oh weird, good catch. Updated!